typed_floats/src/macros.rs

```diff
@@ -179,3 +179,63 @@ macro_rules! assert_float_eq {
         }
     }};
 }
+
+/// Macro to create a constant with a positive value.
+/// The resulting constant will be of type `StrictlyPositiveFinite`.
+/// Will panic at compile time if the value is not a valid.
+///
+/// # Examples
+///
+/// ```
+/// # use typed_floats::*;
+/// const X: StrictlyPositiveFinite = positive_const!(1.0); // f64
+/// const Y: StrictlyPositiveFinite<f64> = positive_const!(f64, 2.0);
+/// const Z: StrictlyPositiveFinite<f32> = positive_const!(f32, 3.0);
+/// ```
+///
+/// Thoses examples will panic at compile time:
+///
+/// ```ignore
+/// # use typed_floats::*;
+/// const X: StrictlyPositiveFinite = positive_const!(0.0);
```

Comment:
```suggestion
/// const invalid: StrictlyPositiveFinite = positive_const!(0.0);
```

Reply:
same for others below...

just reinforcing the idea these will fail

Why?

the further you get from line 196, the less likely you'll know these are counter-examples